src/main/kotlin/de/klg71/keycloakmigration/changeControl/actions/requiredactions/UpdateRequiredActionAction.kt

```diff
@@ -0,0 +1,63 @@
+package de.klg71.keycloakmigration.changeControl.actions.requiredactions
+
+import de.klg71.keycloakmigration.changeControl.actions.Action
+import de.klg71.keycloakmigration.keycloakapi.importFlow
+import de.klg71.keycloakmigration.keycloakapi.model.*
+import java.util.UUID
+
+
+class UpdateRequiredActionAction(
+    realm: String?,
+    private val alias: String,
+    private val providerId: String?,
```

Comment:
I would prefer defining all nullable values with null as default parameter

Reply:
added null default value to default parameters